prow/config/jobs/istio.yaml

```diff
@@ -319,6 +319,22 @@ jobs:
       - name: INTEGRATION_TEST_FLAGS
         value: " --istio.test.retries=1 "
 
+  - name: integ-k8s-129
```

Comment:
We don't need this one. We test, in postsubmit, all versions except the "primary" version. All other tests run with the "primary" version.

Since 1.29 is already the primary version, no need for a redundant job here.

Reply:
Ah good catch; removed